src/applications/reading/index.js

```diff
@@ -128,10 +128,10 @@ function Reading({ application, showSettings }) {
           <div className="reading__input">
             <TextArea
               labelText="Context"
-              cols={50}
```

Comment:
Check removing `cols` completely and see how the rendered view looks? If it is not taking whole space then maybe we can take `cols` property completely.

Reply:
Can this be done at a later time? It works with my change.

Reply:
Will fix later.